_includes/docs/v2.4/required-apis.md

```diff
@@ -12,3 +12,4 @@
 |Cloud SQL|`sql-component.googleapis.com`|
 |Compute Engine|`compute.googleapis.com`|
 |Deployment Manager|`deploymentmanager.googleapis.com`|
+|CloudResource Manager|`cloudresourcemanager.googleapis.com` |
```

Comment:
Move this next to `Resource Manager` and call it `Resource Manager` to be consistent.

Reply:
Removed it as it already exists.